examples/semantic_segmentation/main.py

```diff
@@ -312,7 +312,7 @@ def train(model, model_without_dp, compression_ctrl, train_loader, val_loader, c
         best_miou = resuming_checkpoint['miou']
 
         if "scheduler" in resuming_checkpoint and compression_ctrl.scheduler is not None:
-            compression_ctrl.scheduler.load_state(resuming_checkpoint['scheduler'])
+            compression_ctrl.load_state(resuming_checkpoint['scheduler'])
```

Comment:
Could you open a ticket to rename the 'scheduler' key to the 'compression_controller' key in the checkpoints and link it with 54216.

Reply:
Please see 54427